app.js

```diff
@@ -1,15 +1,16 @@
 const express = require("express");
 const app = express();
-const bodyParser = require("body-parser");
```

Comment:
Why did you remove this from app.js. Isn't it easier to just declare it once?

Reply:
We need to have separate body-parsers for `collect` and the other APIs. `collect` needs to use "raw", while others use JSON, so you need to have two different `app.use(...)`